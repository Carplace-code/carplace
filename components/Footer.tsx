export default function Footer() {
  return (
    <footer className="flex-col items-center justify-between bg-blue-950 px-6 py-4 text-white md:flex">
      <h1 className="text-lg font-bold">CarPlace</h1>
      <p className="mb-2 text-sm">Tu plataforma de confianza para comparar autos usados en Chile.</p>
      <h4 className="text-xs font-semibold">Â© 2025 CarPlace. Todos los Derechos Reservados.</h4>
    </footer>
  );
}
