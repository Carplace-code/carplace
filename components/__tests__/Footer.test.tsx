import { render } from "@testing-library/react";
import { describe, it } from "vitest";

import Footer from "../Footer";

describe("Footer", () => {
  it("renders correctly and matches snapshot", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
