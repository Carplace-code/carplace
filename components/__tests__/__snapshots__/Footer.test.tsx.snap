// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer > renders correctly and matches snapshot 1`] = `
<div>
  <footer
    class="flex-col items-center justify-between bg-blue-950 px-6 py-4 text-white md:flex"
  >
    <h1
      class="text-lg font-bold"
    >
      CarPlace
    </h1>
    <p
      class="mb-2 text-sm"
    >
      Tu plataforma de confianza para comparar autos usados en Chile.
    </p>
    <h4
      class="text-xs font-semibold"
    >
      Â© 2025 CarPlace. Todos los Derechos Reservados.
    </h4>
  </footer>
</div>
`;
